### ISSUE TITLE
Evaluate from pre-trained model state 

### ISSUE THREAD

**Issue by marjanfamili** [alan-turing-institute/affinity-vae#26]
Add ability to select the pre-trained model state to do evaluation from:

This means that if during training the loss divergence we will be able to look at the state of the model in the epoch step where loss was still converging.
